lib/default.nix

```diff
@@ -146,7 +146,7 @@ let
       scrubOptionValue literalExpression literalExample
       showOption showOptionWithDefLocs showFiles
       unknownModule mkOption mkPackageOption mkPackageOptionMD
-      literalMD;
```

Comment:
You didn't pull it from `self.misc`, so this code will break.

Reply:
Thanks for catching. Double-checked now that both forms of mdDoc still work.